backend/docker/docker-compose-cere-mainnet.yml

```diff
@@ -93,8 +93,8 @@ services:
   crawler:
     image: crawler:latest
     build:
-      context: ../
-      dockerfile: ./docker/crawler/Dockerfile
+      context: ../../
+      dockerfile: ./backend/docker/crawler/Dockerfile
```

Comment:
Why did you decide to change paths?

Reply:
because DB migrations that are executed by backend and api modules requires db module that lies outside of them. So CD pipeline were updated to build docker image from parent directory so docker sees db module and can import it.

Reply:
it was discussed before. Also modules structure was in previous PR. That PR was approved by you.

Reply:
Imagine we need to add a new view in our database and expose it via our API. I can add migration to `db` module, and add endpoint in `api`. Once API is re-deployed - db is updated. In your approach, i'll need to redeploy backend.

Reply:
Is it critical to redeploy `backend`? The application was designed that way to run migrations in `backend`:
1. If we add a view then it will be exposed via Hasura GraphQL automatically as we discussed [here](https://github.com/Cerebellum-Network/cerestats/pull/63#discussion_r902684379).
2. If we add a select SQL request then we don't need to migrate the database.
3. If we change DB structure then we need to update the backend.

Reply:
How the deployment script works if we do an update outside `backend` in `db` module? Do we need to rewrite the deployment script?

Reply:
@upalinski @Dmitriy1984Molchanov 
Folks, let's not write novels in comments. 
Migrations have been discussed previously and the solution has been approved. If now we need improvements, share the points in the Slack and let's discuss.